test/functional/wallet_crosschain.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import os
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_raises_rpc_error,
+)
+
+class WalletCrossChain(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 3
+        self.setup_clean_chain = True
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def setup_network(self, split=False):
+        self.setup_nodes()
+
+    def run_test(self):
+        self.log.info("Generating initial blockchains")
+        self.nodes[0].generate(5)
+        self.nodes[1].generate(5)
```

Comment:
In ece5cd111660005ce71cb8a7092f96c3e084d975 "tests: add tests for cross-chain wallet use prevention"

```suggestion
        self.generate(self.nodes[0], 5, sync_fun=self.no_op)
        self.generate(self.nodes[1], 5, sync_fun=self.no_op)
```

Reply:
Removed generation completely, as it doesn't seem necessary for the purposes of this test.